vl-kontrakt-soeknad/soeknad-v3/src/main/resources/xsd/svangerskapspenger/svangerskapspenger-v1.xsd

```diff
@@ -21,11 +21,25 @@
                     <xsd:element name="medlemskap" type="felles:Medlemskap"/>
                     <xsd:element name="opptjening" type="fp:Opptjening" minOccurs="0"/>
                     <xsd:element name="tilretteleggingListe" type="sv:TilretteleggingListe"/>
+                    <xsd:element name="avtaltFerieListe" type="sv:AvtaltFerieListe"/>
                 </xsd:sequence>
                 <xsd:anyAttribute processContents="lax"/>
             </xsd:extension>
         </xsd:complexContent>
     </xsd:complexType>
+    <xsd:complexType name="AvtaltFerieListe">
+        <xsd:sequence>
+            <xsd:element name="avtaltFerie" type="sv:AvtaltFerie" minOccurs="0" maxOccurs="unbounded"/>
+        </xsd:sequence>
+        <xsd:anyAttribute processContents="lax"/>
+    </xsd:complexType>
+    <xsd:complexType name="AvtaltFerie">
+        <xsd:sequence>
+            <xsd:element name="arbeidsgiver" type="sv:Arbeidsgiver"/>
```

Comment:
Blir Arbeidsgiver eller Arbeidsforhold mest riktig her? Hva tenker @AnjaAalerud

Reply:
Søknaden har kanskje ikke et forhold til arbeidsforhold? 

Jeg er ikke så god på xml, og mappingen av ting fra søknad...men det ser ut som om arbeidsgiver, selvsendig næringsdrivene og frilanser mappes til arbeidsforhold i fpsak? i søknadOversetter

Reply:
Tanken med å bruke arbeidsgiver er at det kun spørres om dette for arbeidsforhold av typen virksomhet eller privat arbeidsgiver. Tok den bare i bruk siden den var tilgjengelig. Kaster exception i mapperen i fpsoknad-felles om det kommer fra andre enn dette, så greit for meg om jeg legger det på arbeidsforhold om det er ønskelig. Dette med arbeidsgiver finnes tydeligvis bare i xmlen :)

Reply:
Det er en abstrakt klasse arbeidsforhold med 3 subklasser: frilans, næring og arbeidsgiver. Sistnevnt er abstrakt har igjen to subklasser - Virksomhet og Privat.
Hvis det ikke er naturlig med ferie for SN eller Frilans - så er arbeidsgiver greit. Men hvis man ønsker ferie-lignende opphold for SN eller Frilans (opphold uten ny beregning) så bør man kanskje opp til arbeidsforhold.
Hva sier de faglige om ferie for SN og Frilans i SVP-søknaden? Nå kommer det fra IM (kun for arbeidsgiver)

Reply:
Ifølge Marthe skal vi ikke ha ferie for SN og Frilans. Ferie i svp er jo egentlig ganske ugunstig så for de som styrer selv gir det ikke mening å legge inn ferie.

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/c443c584-2df8-487b-99b2-10816bf9150c">